python/py-pysimdjson/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pysimdjson
+version             6.0.2
+revision            0
+categories-append   devel
+license             MIT
+maintainers         nomaintainer
+
+description         SimdJson bindings for Python
+long_description    {*}${description}
+homepage            https://github.com/TkTech/pysimdjson
+
+checksums           rmd160  f7be364092455f5c98dd59763cddfa0460fc14a7 \
+                    sha256  ddbd6fecd42aa01c5c87d3c79b8ede1885b6763337d21745587a5392572c1f45 \
+                    size    1213207
+
+python.versions     39 310 311 312
+python.pep517_backend \
+                    poetry
```

Comment:
where do you see this uses poetry as backend? I admittedly only looked at upstream's GitHub but see only a setyp.py file there. Perhaps though they do something  special for PyPI - I can't check from my phone.

Reply:
Good question. Perhaps just a copy-paste error, not sure, away from the machine where this was added.
I will remove it now.